expat/xmlwf/xmlmime.c

```diff
@@ -135,6 +135,8 @@ getXMLCharset(const char *buf, char *charset) {
   if (0) {
     if (! matchkey(p, next, "xml") && charset[0] == '\0')
       return;
+  } else {
+    (void)p;
```

Comment:
There is an UNUSED_P() macro which is already used in xmlwf.c, so could also be used here.

...however, it's defined in lib/internal.h, which is not very nice to include from outside the lib. xmlwf.c says `/* for UNUSED_P only */` at the include, but it's easy to miss/forget.

Reply:
@Snild-Sony thanks for having a look! I should note that `xmlmime.c` is one of the unused files that #119 is about, it is not compiled by either of the two build systems today. Let me revert this change and point to #119 right above the list of excluded files in script `apply-clang-tidy.sh` where I'll add `xmlmime.c` for exclusion also. The original idea was to keep the exclusion list as short as possible (and this small change allowed the list to be shorter) but if `xmlmime.c` does not even see a compiler and its warnings, maybe it should be excluded from clang-tidy instead until #119 finds some decision.